tensor2tensor/bin/t2t_trainer.py

```diff
@@ -65,7 +65,7 @@
   flags.DEFINE_string("output_dir", "", "Base output directory for run.")
   flags.DEFINE_string("schedule", "continuous_train_and_eval",
                       "Method of Experiment to run.")
-  flags.DEFINE_integer("eval_steps", 10000,
+  flags.DEFINE_integer("eval_steps", 100,
```

Comment:
Why?

Reply:
Try to reduce the number of log statements on error.

Reply:
What about silencing the end-of-sequence exception? Or what about using 0="the whole dev set" as the default and measuring its size (number of batches) first?

Reply:
> Have a way of measuring its size with all the filtering and bucketing and everything that might be going on?

Yes. Simply iterate through all the data before the first evaluation and remember how many steps were needed. I am no tf.Dataset expert, but it feels strange there is no way to iterate through all the date (one epoch) without this hack with an arbitrarily high eval_steps and a bunch of end-of-sequence exceptions.